src/internal/agent/operations/patch.go

```diff
@@ -0,0 +1,61 @@
+package operations
+
+const (
+	addOperation     = "add"
```

Comment:
We need to decide in a styleguide because we're inconsistent, but uppercase or all caps would be exported, which we don't want here.  Uber has some thoughts on this too:  https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_